scalalib/src/mill/scalalib/TestRunner.scala

```diff
@@ -19,87 +19,103 @@ import scala.collection.mutable
 
 object TestRunner {
   def listClassFiles(base: Path): Iterator[String] = {
-    if (base.isDir) ls.rec(base).toIterator.filter(_.ext == "class").map(_.relativeTo(base).toString)
+    if (base.isDir)
+      ls.rec(base).toIterator.filter(_.ext == "class").map(_.relativeTo(base).toString)
     else {
       val zip = new ZipInputStream(new FileInputStream(base.toIO))
-      Iterator.continually(zip.getNextEntry).takeWhile(_ != null).map(_.getName).filter(_.endsWith(".class"))
+      Iterator
+        .continually(zip.getNextEntry)
+        .takeWhile(_ != null)
+        .map(_.getName)
+        .filter(_.endsWith(".class"))
     }
   }
   def runTests(cl: ClassLoader, framework: Framework, classpath: Agg[Path]) = {
 
-
     val fingerprints = framework.fingerprints()
     val testClasses = classpath.flatMap { base =>
       listClassFiles(base).flatMap { path =>
         val cls = cl.loadClass(path.stripSuffix(".class").replace('/', '.'))
-        fingerprints.find {
-          case f: SubclassFingerprint =>
-
-            (f.isModule == cls.getName.endsWith("$")) &&
-            cl.loadClass(f.superclassName()).isAssignableFrom(cls)
-          case f: AnnotatedFingerprint =>
-            (f.isModule == cls.getName.endsWith("$")) &&
-            cls.isAnnotationPresent(
-              cl.loadClass(f.annotationName()).asInstanceOf[Class[Annotation]]
-            )
-        }.map { f => (cls, f) }
+        fingerprints
+          .find {
+            case f: SubclassFingerprint =>
+              (f.isModule == cls.getName.endsWith("$")) &&
+                cl.loadClass(f.superclassName()).isAssignableFrom(cls)
```

Comment:
Is there a way to get scalafmt to not indent chains of multi-line infix operators?